pages/launcher.vue

```diff
@@ -0,0 +1,1852 @@
+import Avatar from '~/components/ui/Avatar.vue'
+import homepageProjects from '~/generated/homepage.json'
+import LogoAnimated from '~/components/brand/LogoAnimated.vue'
+import Badge from '~/components/ui/Badge.vue'
+import PrismIcon from '~/assets/images/external/prism.svg'
+import ATLauncher from '~/assets/images/external/atlauncher.svg'
+import Checkbox from '~/components/ui/Checkbox.vue'
+
+const val = Math.ceil(homepageProjects.length / 5)
+const os = ref(null)
+
+const rows = shallowRef([
+  homepageProjects.slice(0, val),
+  homepageProjects.slice(val, val * 2),
+  homepageProjects.slice(val * 2, val * 3),
+  homepageProjects.slice(val * 3, val * 4),
+  homepageProjects.slice(val * 4, val * 5),
+])
+
+onMounted(() => {
+  os.value = navigator?.platform.toString()
+})
+</script>
+
+<template>
+  <div>
+    <div class="landing-hero">
+      <h1 class="main-header">
+        Download Modrinth <br />
+        app
+        {{
+          os ? `for ${os?.includes('Mac') ? 'Mac' : os?.contains('Win') ? 'Windows' : 'Linux'}` : ''
+        }}
+      </h1>
+      <h2 class="main-subheader">
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+        labore et dolore magna
+      </h2>
+      <div class="button-group">
+        <nuxt-link to="/mods" class="iconified-button brand-button btn btn-large">
+          More Download Options
+        </nuxt-link>
+        <nuxt-link
+          to="sign-up"
+          class="iconified-button outline-button btn btn-large"
+          rel="noopener nofollow"
+        >
+          Download Modrinth App
+        </nuxt-link>
+      </div>
+      <img src="https://i.imgur.com/n6pheS4.png" alt="cube maze" />
+      <div class="bottom-transition" />
+    </div>
+    <div class="features">
+      <div class="feature-grid">
+        <div class="feature gradient-border mods">
+          <div class="search-bar">
+            <h4>Installed mods</h4>
+            <div class="mini-input">
+              <SearchIcon />
+              <div class="search">Search mods</div>
+            </div>
+          </div>
+          <div class="header row">
+            <div />
+            <div class="cell">Name</div>
+            <div class="cell">Version</div>
+            <div class="cell">Actions</div>
+          </div>
+          <div class="table">
+            <div class="row first">
+              <div class="cell">
+                <Avatar size="sm" src="https://cdn.modrinth.com/data/P7dR8mSH/icon.png" />
+              </div>
+              <div class="cell">
+                <div class="name">Fabric API</div>
+                <div class="description">by modmuss</div>
+              </div>
+              <div class="cell important">1.0.0</div>
+              <div class="cell check">
+                <Checkbox :model-value="true" />
+                <button class="btn icon-only transparent">
+                  <TrashIcon />
+                </button>
```

Comment:
Accessibility: tab navigation interference; these are used purely presentatively and as such, should not be able to receive focus.